package.json

```diff
@@ -1,6 +1,16 @@
 {
-  "name": "visible-website",
+  "name": "visible-wordpress",
   "version": "1.0.0",
+  "scripts": {
+    "export-db": "docker exec $(docker ps -lq) /bin/bash -c \"sudo -u www-data wp db export /data/database.sql\"",
+    "import-db": "docker exec $(docker ps -lq) /bin/bash -c \"sudo -u www-data wp db import /data/database.sql\"",
+    "update-plugins": "docker exec $(docker ps -lq) /bin/bash -c \"sudo -u www-data wp plugin update --all\"",
+    "update-themes": "docker exec $(docker ps -lq) /bin/bash -c \"sudo -u www-data wp theme update --all\"",
+    "plugin-install": "docker exec $(docker ps -lq) /bin/bash -c \"sudo -u www-data wp plugin install $PLUGIN\"",
+    "plugin-update": "docker exec $(docker ps -lq) /bin/bash -c \"sudo -u www-data wp plugin update $PLUGIN\"",
```

Comment:
For lines 9-10 to work correctly, they need to be ran like this:

(Assuming that `some-plugin` is the one we're interested in installing)
`PLUGIN=some-plugin npm run plugin-install`